firebase/src/main/backend/firestore-v3/FirestoreCollectionV3.ts

```diff
@@ -537,7 +537,7 @@ export class FirestoreCollectionV3<Proto extends DBProto<any>>
 		if (index === -1)
 			throw HttpCodes._4XX.BAD_REQUEST('Invalid Object Version', `Provided item with version(${version}) which doesn't exist for collection '${this.dbDef.dbKey} (${__stringify(this.dbDef.versions)})' `);
 
-		return index !== versions.length - 1;
```

Comment:
did you make sure this wont return true on other cases, we should turn on a lower log level to make sure this is not triggered unnecessarily